runtime/polkadot/src/lib.rs

```diff
@@ -113,6 +125,9 @@ impl Filter<Call> for BaseFilter {
 			Call::DummyPurchase(_) =>
 				false,
 
+			// More Parachains stuff
+			Call::Configuration(_) | Call::Inclusion(_) | Call::Initializer(_) | Call::Paras(_) | Call::ParaScheduler(_) => false,
```

Comment:
I'm guessing that the parachains modules should not be callable, but I'm not certain, and would appreciate :+1: or :-1:  from someone who knows.